### PR TITLE
chore/sample angular app

### PR DESCRIPTION
**Pull request by Jidapa-Pai**
## Description
Angular Sample App for V5

### CODE REVIEW

File: samples/angular/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "angular-app",
```
Comment (Theeraphat-Sorasetsakul) [inline samples/angular/package.json:2]: I think app name should be "angular-sample".

File: samples/angular/src/app/app.component.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    this.checked = defaultTheme === 'dark';
+  }
+
+  ngAfterViewInit (): void {
+    this.generateChart();
+  }
+  
+  async loadTheme (theme: string): Promise<unknown> {
+    document.documentElement.setAttribute('theme', theme);
+    document.documentElement.style.setProperty('--theme', theme);
```
Comment (Theeraphat-Sorasetsakul) [inline samples/angular/src/app/app.component.ts:45]: Do we need this line?

File: samples/angular/src/app/components/profile-form/profile-form.component.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      id="name"
+      class="form-field"
+      formControlName="name"
+      [error]="name && name.invalid && (name.dirty || name.touched)"
+      ngDefaultControl></ef-text-field>
+    </div>
+    <div class="form-input">
+      <label id="gender_group">Gender</label>
+      <div role="radiogroup" aria-labelledby="gender_group" #genderGroup>
+        <ef-radio-button name="gender" value="male" [checked]="gender === 'male'? 'true': undefined">Male</ef-radio-button>
```
Comment (Theeraphat-Sorasetsakul) [inline samples/angular/src/app/components/profile-form/profile-form.component.html:15]: Do we need to bind back to Angular? I think checked property do correctly without this.

File: samples/angular/src/app/app.component.ts (hunk 2 above)
Comment (Jidapa-Pai) [reply samples/angular/src/app/app.component.ts:45]: We can remove these lines.

Comment (Jidapa-Pai) [reply samples/angular/src/app/app.component.ts:45]: Actually, we can remove only line 45. Line 44 is require for theme switching.

File: samples/angular/src/app/components/profile-form/profile-form.component.html (hunk 3 above)
Comment (Jidapa-Pai) [reply samples/angular/src/app/components/profile-form/profile-form.component.html:15]: Agree, I will remove it.

File: samples/angular/.browserslistrc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+last 1 Chrome version
+last 1 Firefox version
+last 2 Edge major versions
+last 2 Safari major versions
+last 2 iOS major versions
+Firefox ESR
```
Comment (wsuwt) [inline samples/angular/.browserslistrc:16]: We should make sure this app also runs on legacy browsers.

File: samples/angular/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Running end-to-end tests
+
+Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To use this command, you need to first add a package that implements end-to-end testing capabilities.
+
+## Further help
+
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
```
Comment (wsuwt) [inline samples/angular/README.md:27]: At minimum, README should include how to scaffold, start project and test project. I would avoid Angular's default readme, I don't think we want to promote installing `@angular/cli` as a global package.

**Comment by Jidapa-Pai** [comment]
Move to https://github.com/Refinitiv/refinitiv-ui/pull/150

**Pull request closed without merging**